README.md

```diff
@@ -1,33 +1,13 @@
 # BibMine - Bibliographic analysis pipeline
-[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/ajdapretnar/BibMine/master?filepath=main.ipynb)
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/ajdapretnar/BibMine/fix_structure?filepath=main.ipynb)
```

Comment:
Do we want to pin the badge to this branch instead of master?

Reply:
Nope, that was for testing only. Will fix it.